### PR TITLE
database backup service

### PR DESCRIPTION
**Pull request by GrantBirki**
This pull request adds a new service to the docker-compose stack. It is a simple nodejs script that does the following:

1. starts a cron job that runs at 12pm and 1 am (about 12ish hours a part) - twice a day
2. connects to the `database` container and does a full mysql dump
3. compresses the mysql dump with `gzip`
4. uploads the compressed database dump to Azure blob storage under the `Cold` storage tier

All backups are versioned and have a lifecycle management policy to be kept for 7 days before they are soft deleted.

**TL;DR**: We now have a service that backs up our database twice a day. Those backups are stored in Azure for 1 week.

### CODE REVIEW

**Comment by GrantBirki** [comment]
.deploy

**Comment by GrantBirki** [comment]
.unlock

**Comment by GrantBirki** [comment]
.deploy

**Comment by GrantBirki** [comment]
.deploy

**Comment by GrantBirki** [comment]
.deploy

**Review by Razzmatazzz — APPROVED** [review]

**Comment by GrantBirki** [comment]
.wcid
